kokoro/build_binaries_oak_containers.sh

```diff
@@ -17,28 +16,37 @@ cd "$(dirname "$0")/.."
 ./scripts/docker_pull
 ./scripts/docker_run nix develop .#containers --command just kokoro_oak_containers
 
-mkdir -p "$KOKORO_ARTIFACTS_DIR/test_logs/"
+mkdir -p "${KOKORO_ARTIFACTS_DIR}/test_logs/"
 cp --preserve=timestamps \
     ./target/nextest/default/*.xml \
     "${KOKORO_ARTIFACTS_DIR}/test_logs/" || true
 
-mkdir -p "$KOKORO_ARTIFACTS_DIR/binaries/"
+mkdir -p "${KOKORO_ARTIFACTS_DIR}/binaries/"
 
-# Store the git commit hash in the name of an empty file, so that it can be efficiently found via a glob.
-touch "$KOKORO_ARTIFACTS_DIR/binaries/git_commit_${KOKORO_GIT_COMMIT_oak:?}"
+# Store the git commit hash in the name of an empty file, so that it can
+# be efficiently found via a glob.
+touch "${KOKORO_ARTIFACTS_DIR}/binaries/git_commit_${KOKORO_GIT_COMMIT_oak:?}"
 
 # Copy the generated binaries to Placer. The timestamps are used to convey
 # the creation time.
-export GENERATED_BINARIES=(
+readonly generated_binaries=(
     ./target/stage1.cpio
-    ./oak_containers_kernel/target/vmlinux
     ./oak_containers_kernel/target/bzImage
     ./oak_containers_system_image/target/image.tar.xz
     ./oak_containers_hello_world_container/target/oak_container_example_oci_filesystem_bundle.tar
     ./oak_functions_containers_container/target/oak_functions_container_oci_filesystem_bundle.tar
 )
-cp --preserve=timestamps \
-    "${GENERATED_BINARIES[@]}" \
-    "${KOKORO_ARTIFACTS_DIR}/binaries/"
+readonly binary_names=(
```

Comment:
Why can we not rename the actual generated files? adding a level of indirection seems to just give additional names to the same underlying object, i.e. make things more ambiguous. IMO we should decide whether to keep extensions or not, and then stick with it even internally, and also unify the actual names. WDYT?

Reply:
But why is that necessary? Is it because we don't want the extension of the filename to end up in the folder name when imported?

> I think in general, the binary_name is a unique identifier to refer to a binary package. It is managed by Transparent Release. The name of the binary file on disk is a different thing; it is managed at the source.

Yes but why do we need to manage differently? It seems we should just agree on one name and use it in both places. Or is that not technically possible?

Reply:
Hypothetical but close enough to current reality: Assume we have bzImage4, bzImage8, bzImage16 generated from the same GitHub source project (kernels optimized for 4,8,16 GB of RAM, whatever). We want to pick bzImage8 as oak_linux_kernel. The source project is just about making Linux kernels, it doesn't care how its output is used. Pulling the name through would require to go back to the source project and rename outputs to bzImage4, oak_linux_kernel, bzImage16. The semantics is lost, and another package needing bzImage8 will be be confused. This seems unstable.